cms/djangoapps/contentstore/views/course.py

```diff
@@ -593,28 +626,48 @@ def _remove_in_process_courses(courses, in_process_course_actions):
     removes any in-process courses in courses list. in-process actually refers to courses
     that are in the process of being generated for re-run
     """
+
     def format_course_for_view(course):
         """
         Return a dict of the data which the view requires for each course
         """
-        return {
-            'display_name': course.display_name,
-            'course_key': unicode(course.location.course_key),
-            'url': reverse_course_url('course_handler', course.id),
-            'lms_link': get_lms_link_for_item(course.location),
-            'rerun_link': _get_rerun_link_for_item(course.id),
-            'org': course.display_org_with_default,
-            'number': course.display_number_with_default,
-            'run': course.location.run
-        }
+        if is_course_summary(course):
+            course_id = course['locator']
+            return {
+                'display_name': course['display_name'],
+                'course_key': unicode(course_id),
+                'url': reverse_course_url('course_handler', course_id),
+                'lms_link': get_lms_link_for_item(course['location']),
+                'rerun_link': _get_rerun_link_for_item(course_id),
+                'org': course_id.org,
```

Comment:
Adam, it looks like `display_org_with_default` is a property method in `CourseDescriptor`. To able to use this method we need to load to course.  What do you think we can do here? 

Do we have to look for `display_organization` attrib in the XML as well as `org`? 
Thoughts?